dashboard/src/components/AppView/ResourceTable/ResourceItem/SecretItem/SecretItemDatum.v2.scss

```diff
@@ -21,3 +21,7 @@
     outline: none;
   }
 }
+
+.secret-datum-content {
+  border-bottom: 1px solid var(--clr-forms-border-color, #b3b3b3) !important;
```

Comment:
I needed this because by default, `readOnly` input fields don't have the border-bottom but I wanted it to show the length of the input field.

Reply:
Interesting!